nss-day-cohort-48/rare-server-wanderering-wizards
# Ajw tag a post

#### Changes Made

1. Modified file `posts/request.py` to include `tags`.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ajw-tag-a-post
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Insert some columns into the PostTags table and check if the tags are connected to the correct posts in Postman
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
